
string human_id
string[] object_ids
sensor_msgs/PointCloud2[] attentions